pkg/operator/starter.go

```diff
@@ -104,6 +112,37 @@ func RunOperator(clientConfig *rest.Config, stopCh <-chan struct{}) error {
 	return fmt.Errorf("stopped")
 }
 
+func StaticPodIsRunningFunc(podlister corev1listers.PodLister, ns string) installer.StaticPodInstallStateFunc {
+	return func(installerPod *corev1.Pod, deploymentID int32, nodeName string) (installer.StaticPodInstallState, error) {
+		switch installerPod.Status.Phase {
```

Comment:
It seems weird to pass the installerPod here.  I'm not seeing how that affects whether a particular deployment's status